include/device.h

```diff
@@ -6,14 +6,35 @@ namespace dronecore {
 
 class DeviceImpl;
 
+/**
+ * A Device represents a unit such as a drone or robot.
+ *
+ * A device can consists of multiple components such as an autopilot with a gimbal and camera.
+ * The device class allows to access plugins such as Action or Telemetry because it is based
```

Comment:
When you added `DroneCore::device()`. before it got turned into a real link in rendered HTML -  might be worth doing the same here with Action and Telemetry.

Reply:
Done in 8e83ca1.